model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -2508,7 +2508,7 @@ async def execute(
         job_id = await self.docker_image_batch_job_gateway.create_docker_image_batch_job(
             created_by=user.user_id,
             owner=user.team_id,
-            job_config=engine_request.dict(),
+            job_config=engine_request.model_dump(by_alias=True),
```

Comment:
what exactly is broken with the current code?

Reply:
added to description